package.json

```diff
@@ -108,5 +110,16 @@
     "rollup-plugin-visualizer": "0.6.0",
     "rxjs": "6.2.1",
     "styled-components": "3.3.2"
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "eslint --fix",
+      "git add"
+    ]
+  },
+  "husky": {
```

Comment:
you may write in the npm scripts, like:
```json
{
  "scripts": {
    "precommit": "lint-staged",
  }
}
```

Reply:
seems like `husky/@next` api changed to this one